source/MaterialXCore/Value.cpp

```diff
@@ -254,6 +254,60 @@ ScopedFloatFormatting::~ScopedFloatFormatting()
     Value::setFloatPrecision(_precision);
 }
 
+StringVec parseStructValueString(const string& str)
```

Comment:
We should move this function definition into the `Global functions` block above, in order to make it more straightforward to find.

Reply:
done